blueprint/security/hybrid-system-security-plan.md

```diff
@@ -916,31 +944,28 @@ Local sites and data stores | On-premises SharePoint Server
 
 System patching of Office 365 and Azure AD are not applicable as these cloud components are a Microsoft responsibility.
 
-System patching of endpoint devices is required, and this is accomplished via Microsoft Endpoint Manager - Intune (Intune) or MECM policies setting the frequency, installation options and reporting values.
+System patching of endpoint devices is required, and this is accomplished via Intune or MECM policies setting the frequency, installation options and reporting values.
 
 #### HybridSystem compliance approach
 
 The HybridSystem compliance approach is to primarily utilise native cloud capabilities that are patched as part of the service. 
 
-For patching endpoint devices deployed by the Agency, the HybridSystem provides configuration of which patches to apply, deferral periods, update behaviour and reporting.
+The HybridSystem uses Intune or MECM to automatically deploy operating system, application, driver, and firmware (where supported) updates to Windows 10 SOE devices. 
+
+The blueprint does not include third-party applications or any unsupported software.
 
 #### Security controls provided by the HybridSystem
 
-* The HybridSystem provides the types of updates that are applied to endpoints.
-* The HybridSystem configures the intervals for checking for new updates.
-* The HybridSystem provides the reporting of device status to determine which devices have received updates.
 * The HybridSystem includes a system administration SOP which specifically references patching.
-* All configurations are included in the relevant ABAC.
-* Application and driver patches will be automatically deployed via Intune or MECM for the HybridSystem Windows 10 SOE.
-* Operating system patches will be automatically deployed via Intune or MECM for the HybridSystem Windows 10 SOE.
+* Intune or MECM is configured to automatically install updates within 48 hours on all Windows 10 devices.
 * Intune or MECM provides a centralised and managed approach to patching.
 * Windows Update verifies the integrity of patches before installing them.
+* Microsoft Defender for Endpoint provides a continuous vulnerability management capability for all Windows 10 devices.
 
 #### Residual controls to be addressed by the Agency
 
-* The Agency is responsible for system patching for all systems deployed within the HybridSystem.
-* The Agency is responsible for any firmware patching dependent on the specific hardware model chosen by them.
-* The Agency is responsible for viewing the reporting and alerts and rectification of faults as they occur.
+* The Agency is responsible for maintaining and auditing a software register.
+* The Agency is responsible for patching any third-party applications  deployed to Windows 10 devices.
```

Comment:
seems to be a double space in beteen 'applications' and 'deployed'

Reply:
Fixed - along with some others I found in both SSPs.